src/popup/components/UrlStatus.vue

```diff
@@ -5,13 +5,13 @@
     :class="status"
   >
     <span class="title">{{ statusData.title }}</span>
-    <a
+    <BtnBase
```

Comment:
In my opinion BtnBase component should not be used in regular components. It exists to create other button-like components. Maybe you can use `BtnMain` or `BtnPlain` here?